assets/js/selectWoo/selectWoo.js

```diff
@@ -5426,17 +5436,35 @@ S2.define('select2/core',[
 
           evt.preventDefault();
         }
-      } else {
+
+        // Move the focus to the selected element on keyboard navigation.
+        // Required for screen readers to work properly.
+        if (key === KEYS.DOWN || key === KEYS.UP) {
+            self.focusOnActiveElement();
+        } else {
+          // Focus on the search if user starts typing.
+          $('.select2-search__field').focus();
```

Comment:
Need to prevent this from focusing on other search fields if multiple selects are on the page. Small tweak.